src/main/java/org/verdictdb/connection/InMemoryAggregate.java

```diff
@@ -80,13 +85,13 @@ public static void createTable(DbmsQueryResult dbmsQueryResult, String tableName
     statement.executeBatch();
   }
 
-  public static DbmsQueryResult executeQuery(SelectQuery query) throws VerdictDBException, SQLException {
+  public DbmsQueryResult executeQuery(SelectQuery query) throws VerdictDBException, SQLException {
     String sql = selectQueryToSql.toSql(query).toUpperCase();
     ResultSet rs = conn.createStatement().executeQuery(sql);
     return new JdbcQueryResult(rs);
   }
 
-  public static String combinedTableName(String combineTableName, String targetTableName, SelectQuery dependentQuery)
+  public String combinedTableName(String combineTableName, String targetTableName, SelectQuery dependentQuery)
```

Comment:
Please change the function name to "combineTables()", change the first argument to "combinedTableName", and change the second argument to "newAggTableName".

Reply:
On line 101, the statement must be closed.

Reply:
On line 137, the statement must be closed.